x-pack/plugins/secops/public/components/timeline/body/renderers/plain_column_renderer.test.tsx

```diff
@@ -55,11 +58,13 @@ describe('plain_column_renderer', () => {
     );
     const wrapper = mount(
       <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
-        <ReduxStoreProvider store={store}>
-          <DragDropContext onDragEnd={noop}>
-            <span>{column}</span>
-          </DragDropContext>
-        </ReduxStoreProvider>
+        <KibanaConfigContext.Provider value={mockFramework}>
+          <ReduxStoreProvider store={store}>
+            <DragDropContext onDragEnd={noop}>
```

Comment:
Would it make sense to write a `TestProvider` that provides all contexts to clean up tests/make them more easily readable?

Reply:
Was thinking the same thing. We should at least add a tech debt ticket for it or feel free to add one to this PR and I will use it.

Reply:
Created https://github.com/elastic/ingest-dev/issues/308 to track since this PR is just about good to go.